### PR TITLE
Fix cross-ref warnings

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>The provider for the <see cref="T:Microsoft.Extensions.Logging.EventSource.EventSourceLogger" />.</summary>
```
Comment (mairaw) [inline xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml:33]: Maybe keep the description here but remove the link?
EventSourceLogger is internal: https://github.com/aspnet/Logging/blob/master/src/Microsoft.Extensions.Logging.EventSource/EventSourceLogger.cs

Comment (gewarren) [reply xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml:33]: I originally did that, but then realized that the summary is not adding anything that the name of the type doesn't already tell you.

Comment (BillWagner) [reply xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml:33]: I'd agree with @gewarren here.

**Review by BillWagner — APPROVED** [review]
This looks great @gewarren 

Go ahead and :shipit:

**Comment by gewarren** [comment]
Thanks Bill and Maira!

File: xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml (hunk 1 above)
Comment (mairaw) [reply xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml:33]: You probably don't have compliance requirements here so I guess it's fine.

**Pull request merged**